### ISSUE TITLE
[NL - Amsterdam] - Backbase B.V. - Principal iOS Engineer (Mobile SDK)

### ISSUE THREAD

**Issue by arthurpalves** [CocoaHeadsBrasil/vagas#773]
## Backbase
Backbase is disrupting the banking industry. Worldwide, renowned banks and financial institutions build their businesses using our mobile apps. What we do impacts the banking experience of millions of users every single day.

Whether or not you’ve heard of us, you and your friends have probably used some of the products we’ve created. Backbase accelerates the transformation of digital banking. From being a Dutch startup company five years ago, we are now an international market leader. Despite having a leading market position, we haven’t lost our startup mentality and we’re not done innovating just yet. Working in teams with highly talented and skilled professionals from all over the world, we share a drive to make things better, nicer, and more advanced. More than 400 specialists dedicate themselves to what we do best, which is transforming global financial services with inspiring, innovative solutions. At Backbase, we love to create outstanding experiences, not just for our customers and users, but for ourselves, and this is what motivates us to transform our industry by delivering our best work.

## Position
Principal iOS Engineer (Mobile SDK).

## Job Type
Permanent.

## As a Principal iOS engineer...

**TL;DR:** You're a freaking rock star 🤘🤩🎸 

You will design, build and extend core capabilities for the Backbase Mobile Platform from tooling to client-side libraries and security features used in all of the Backbase-powered mobile applications. We love clean, testable, and maintainable code as we are building a product that is continuously evolving.

You will be part of a small multidisciplinary agile team that is responsible for delivering an end-to-end solution, from concept to production-ready software.

You deliver high-quality, well-designed code that is elegant, clean and is easy to maintain for others.

You are used to addressing complex business questions and how to successfully translate these into technical designs and -implementations. Within your agile team you coach your peers to learn the same skills.

As a Principal iOS Engineer, you will help the team to make technical decisions. Your peers in the team look to you to provide the final word on all matters iOS.

As a team member, you will contribute to all aspects of product development, starting from gathering feedback & requirements and implementing actual functionality, to writing tests, documentation and supporting adoption.

In our teams, we use the following technologies:
* Objective-C / Swift / Python / Ruby
* Android: Java Kotlin
* Jenkins + Groovy

You are familiar with clean coding, the SOLID principles and know your design patterns and how to apply them in real life. You are aware of the importance of a well-designed architecture. You are not afraid of proposing new designs and get into discussions with your peers about new ways of solving problems. We like to keep work interesting and we encourage our developers to develop their personal strengths. Whether it’s deep-diving into technology, Agile processes, or moving into full-stack development, at Backbase we offer you the room to grow.

###### Required
* Fluency in English.
*  8+ years of relevant working experience working on complex iOS-based products;
* You are accustomed to translating high-level business requirements into technical designs and coach your peers to do the same;
* You care about code quality and you thrive in a complex domain;
* Security and other non-functionals don’t scare you and you have what it takes to uphold our high standards;
* You have technical experience with architecture patterns like MVVM, VIPER or MVP;
* Strong belief that writing clean, maintainable and scalable code is the best way to work;
* Having an understanding of the SDLC and experience with Continuous Delivery tools would be a big plus;
* You work and play well with others as you are part of an Agile team.
* Did we already say you need to be a good listener?
* You are excellent in reading code in order to be able to analyse quickly.

## Benefits
* Market rate salary
We work hard, play hard, and earn hard

* Visa sponsorship & relocation package
For you, your partner, children and pets

* Multicultural environment
Work and play with people from around the world

* Contributory pension scheme
We’ll help you fill your coffers for old age

* Location
We work in beautiful Amsterdam. Nuff said, right?

* 25 days annual leave
Our full-timers know how to cut loose and recharge

* Company fitness plan
Burn off your free lunch and reenergise!

* Social events
Beach, boat, beers and board games. We do the lot!

* Free lunch & refreshments
Stay healthy and focused with food, drinks and good company

## Who you will be working with
You will work with hundreds of people from all around the world, from 60+ nationalities, but with 32+ Brazilians 🇧🇷 in our Amsterdam office alone you won't get rid of Portuguese so easily.

Take a look at a small bit of our life here: https://www.instagram.com/life_at_backbase

## Contact me ;)
arthura@backbase.com
